<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSPIndicators</class>
 <widget class="QWidget" name="RSPIndicators">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Радиолокационная система посадки</string>
  </property>
  <widget class="MainLocator" name="FirstIndicator" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="EquiangularTriangleLocator" name="SecondIndicator" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="RightTriangleLocator" name="ThirdIndicator" native="true">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>20</width>
     <height>441</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>20</width>
     <height>441</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>443</y>
     <width>1321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>450</y>
     <width>16</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="DRLGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>ДРЛ</string>
   </property>
   <widget class="QLabel" name="LabelRangeMarks">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>64</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отметки дальности:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectRangeMarks">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>62</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectAzimuthMarks">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>25</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAzimuthMarks">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>27</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отметки азимута:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectScale">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelScale">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBrightness">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Яркость дисплея:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelDisplayLightning">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Подсветка:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeDisplayLightning">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>162</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorVARU">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>221</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelFocus">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Фокус:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelVARU">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ВАРУ:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorFocus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>192</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorBrightness">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>132</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeTrashIntensity">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>44</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTrashIntensity">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>26</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Усиление приёмника:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectWorkVariant">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>80</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelWorkVariant">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим работы:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLocatorState">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeViewStateAll">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Все отметки</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckActiveNoiseShow">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Шумовая помеха</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckActiveAnswerShow">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>131</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ответно-импульсная</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>20</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckActiveAnswerShow_2">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Несинхронно-
импульсная</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="PRLGroup">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>601</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>ПРЛ</string>
   </property>
   <widget class="QSlider" name="ChangeTrashIntensityEquiangular">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>91</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorFocusEquiangular">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBrightnessEquiangular">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>118</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Яркость дисплея:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectScaleEquiangular">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelFocusEquiangular">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>148</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Фокус:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeViewStateAllEquiangular">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Отобразить все скрытые метки</string>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorBrightnessEquiangular">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectRangeMarksEquiangular">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRangeMarksEquiangular">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отметки дальности:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLocatorStateEquiangular">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelScaleEquiangular">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTrashIntensityEquiangular">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>91</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Интенсивность:</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>21</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLocatorStateRightTriangle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorFocusRightTriangle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRangeMarksRightTriangle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>32</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отметки дальности:</string>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeTrashIntensityRightTriangle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>91</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectScaleRightTriangle">
    <property name="geometry">
     <rect>
      <x>447</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBrightnessRightTriangle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>118</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Яркость дисплея:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeViewStateAllRightTriangle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Отобразить все скрытые метки</string>
    </property>
   </widget>
   <widget class="QSlider" name="ChangeIndicatorBrightnessRightTriangle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelScaleRightTriangle">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectRangeMarksRightTriangle">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTrashIntensityRightTriangle">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>91</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Интенсивность:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelFocusRightTriangle">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>148</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Фокус:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonTargetsSettings">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>480</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelTargetsSettings">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>480</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки целей:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeTargetsState">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>510</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Запуск целей</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainLocator</class>
   <extends>QWidget</extends>
   <header>mainlocator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EquiangularTriangleLocator</class>
   <extends>QWidget</extends>
   <header>equiangulartrianglelocator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RightTriangleLocator</class>
   <extends>QWidget</extends>
   <header>righttrianglelocator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
